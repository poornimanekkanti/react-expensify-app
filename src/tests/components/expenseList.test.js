import React from 'react'
import { shallow } from 'enzyme'
import { ExpenseList } from '../../components/ExpenseList'
import expenses from '../fixtures/expenses'

test('should render ExpenseList with expenses', () => {
  const wrapper = <ExpenseList expenses={expenses} />
  expect(wrapper).toMatchSnapshot()
})

test('should render ExpenseList with empty message', () => {
  const wrapper = <ExpenseList expenses={[]} />
  expect(wrapper).toMatchSnapshot()
})
