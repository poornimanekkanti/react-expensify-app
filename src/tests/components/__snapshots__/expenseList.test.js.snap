// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList with empty message 1`] = `
<Unknown
  expenses={Array []}
/>
`;

exports[`should render ExpenseList with expenses 1`] = `
<Unknown
  expenses={
    Array [
      Object {
        "amount": 195,
        "createdAt": 0,
        "description": "Gum",
        "id": "1",
        "note": "",
      },
      Object {
        "amount": 109500,
        "createdAt": -345600000,
        "description": "Rent",
        "id": "2",
        "note": "",
      },
      Object {
        "amount": 4500,
        "createdAt": 345600000,
        "description": "Credit Card",
        "id": "3",
        "note": "",
      },
    ]
  }
/>
`;
